docs/user-guide/installation.rst

```diff
@@ -9,42 +9,185 @@ Installation
+
+            $ sudo apt update
+            $ sudo apt install python3-dev python3-pip
+            $ sudo pip3 install -U virtualenv
+
+    .. tab:: macOS
+
+        Install the `Homebrew <https://brew.sh>`_ package manager if you haven't already.
+
+        Once you're done, you can install Python and virtualenv.
+
+        .. code-block:: bash
+
+            $ brew update
+            $ brew install python  # Python 3
+            $ sudo pip3 install -U virtualenv  # system-wide install
+
+    .. tab:: Windows
+
+        .. raw:: html
+
+            Make sure the Microsoft VC++ Compiler is installed, so python can compile
+            any dependencies. You can get the compiler from <a class="reference external"
+            href="https://visualstudio.microsoft.com/visual-cpp-build-tools/"
+            target="_blank">Visual Studio</a>. Download the installer and select
+            VC++ Build tools in the list.
+
+        Install `Python 3 <https://www.python.org/downloads/windows/>`_ (64-bit version) for Windows.
+
+        .. code-block:: bat
+
+            C:\> pip3 install -U pip virtualenv
+
+
+2. Create a virtual environment (strongly recommended)
+------------------------------------------------------
+
+Tools like `virtualenv <https://virtualenv.pypa.io/en/latest/>`_ and `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_ provide isolated Python environments, which are cleaner than installing packages systemwide. They also let you install packages without root privileges.
+
+.. tabs::
+
+    .. tab:: Ubuntu / macOS
+
+        Create a new virtual environment by choosing a Python interpreter and making a ``./venv`` directory to hold it:
+
+        .. code-block:: bash
+
+            $ virtualenv --system-site-packages -p python3 ./venv
+
+        Activate the virtual environment:
+
+        .. code-block:: bash
+
+            $ source ./venv/bin/activate
 
-Windows Prerequisites
-~~~~~~~~~~~~~~~~~~~~~
+    .. tab:: Windows
 
-.. raw:: html
+        Create a new virtual environment by choosing a Python interpreter and making a ``.\venv`` directory to hold it:
 
-     Make sure the Microsoft VC++ Compiler is installed, so python can compile
-     any dependencies. You can get the compiler from <a class="reference external"
-     href="https://visualstudio.microsoft.com/visual-cpp-build-tools/"
-     target="_blank">Visual Studio</a>. Download the installer and select
-     VC++ Build tools in the list.
+        .. code-block:: bat
 
+            C:\> virtualenv --system-site-packages -p python3 ./venv
+
+        Activate the virtual environment:
+
+        .. code-block:: bat
+
+            C:\> .\venv\Scripts\activate
+
+
+3. Install Rasa and Rasa X
+--------------------------
+
+.. tabs::
+
+    .. tab:: Inside a virtualenv
+
+        To install both Rasa and Rasa X in one go:
+
+        .. code-block:: bash
+
+            (venv) $ pip install rasa-x --extra-index-url https://pypi.rasa.com/simple
```

Comment:
isn't this slightly confusing? I suspect people might think they have to copy-paste the `(venv) $` part